contracts/treasury/test/treasury_test.ts

```diff
@@ -17,8 +17,9 @@ import {
     randomAddress,
     verifyEventsFromLogs,
 } from '@0x/contracts-test-utils';
-import { BigNumber } from '@0x/utils';
-import * as _ from 'lodash';
+import { Vote } from '@0x/protocol-utils/lib/src/votes';
```

Comment:
So what you need to do is export `Vote` from `src/index.ts` in `protocol-utils` so you can just do
```suggestion
import { Vote } from '@0x/protocol-utils';
```